includes/commands/class-mu-migration-base.php

```diff
@@ -42,7 +42,7 @@ protected function process_args( $default_args = array(), $args = array(), $defa
 	 */
 	protected function all_posts( $query_args, $callback, $verbose = true ) {
 		if ( ! is_callable( $callback ) ) {
-			self::error( __( "The provided callback is invalid", 'mu-migration' ) );
+			WP_CLI::error( __( "The provided callback is invalid", 'mu-migration' ) );
```

Comment:
Since there is no `error()` method in this class, I changed this to use `WP_CLI::error()`.

I thought of adding an `error()` method to also allow passing of the `$verbose` parameter, but then thought it might not be necessary for an error output.

@nicholasio What's your view on this one?